### DISCUSSION TITLE
Tesla A10 and cuBLAS and cuDNN 8.x libs versions

### DISCUSSION THREAD

**Discussion by 1001ruchka** [Purfview/whisper-standalone-win#197] [General]
It turned out that depending on the version of cuBLAS and cuDNN 8.x libs - errors may appear or disappear on Tesla a10.

Version 4
Timing 02:01:56,430 - ... instead of text
[Purfview-Whisper-Faster log-4.zip](https://github.com/Purfview/whisper-standalone-win/files/14331365/Purfview-Whisper-Faster.log-4.zip)

Version 2
Timing 02:01:01,750 - Speech is normally recognized
[Purfview-Whisper-Faster log-5.zip](https://github.com/Purfview/whisper-standalone-win/files/14331393/Purfview-Whisper-Faster.log-5.zip)

At the same time, if you compare the quality of recognition of text not in English (I used Russian) - the recognition quality is higher by v4

Command line arguments: --sentence --compute_type=float16
I gave up using int8 with them Whisper makes huge long sentences. float32 is slower than float16

Whisper-Faster r172.3 version was used.

**Comment by Purfview** [comment]
What is "Version 4.7" & "Version 2.7"?

**Comment by 1001ruchka** [comment]
![image](https://github.com/Purfview/whisper-standalone-win/assets/5934321/e6df8f06-1fa3-466f-b434-c6277e4ed26c)
https://github.com/Purfview/whisper-standalone-win/releases/tag/libs

**Reply by Purfview** [reply]
`.7z` it's the file extension aka 7zip. 😆

**Reply by 1001ruchka** [reply]
Heh. I was confused by the zip file. So it's v2 and v4

**Reply by Purfview** [reply]
"zip" there is for SE downloads as it can't unpack 7z.

**Comment by Purfview** [comment]
> Timing 02:01:56,430 - ... instead of text

That's probably an hallucination from custom initial prompt, I'll change it in the next version.
Try `-prompt None`, but then anti "no-punctuation" stuff won't work.

**Reply by Purfview** [reply]
Actually, try `-prompt auto`, I'll change to that by default.

**Reply by 1001ruchka** [reply]
It all seems to have worked out. V4
[Purfview-Whisper-Faster log no promt v4.zip](https://github.com/Purfview/whisper-standalone-win/files/14331962/Purfview-Whisper-Faster.log.no.promt.v4.zip)

**Reply by 1001ruchka** [reply]
I have a hypothesis as to why there might have been a problem. 1 phrase in this audio was not in English. Whisper translated it into English, but the phrase itself was in Russian. At the same time 90% of the audio is in English. Perhaps this first phrase was causing this glitch.

**Reply by Purfview** [reply]
I how it's with `auto`?

**Reply by 1001ruchka** [reply]
[Purfview-Whisper-Faster log no promt v4.zip](https://github.com/Purfview/whisper-standalone-win/files/14331962/Purfview-Whisper-Faster.log.no.promt.v4.zip)
Just what is done with the auto parameter.

**Reply by Purfview** [reply]
Oh, I thought that you used `None` as you named it as "no promt".
